src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: New Relic discussions and commenting integration with Slack
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+metaDescription: How to use our new collaboration features with Slack
+---
+import post_on_slack from 'images/post_on_slack.png'
+
+import slack_app_management from 'images/slack_app_management.png'
+
+import slack_auth from 'images/slack_auth.png'
+
+import slack_synced from 'images/slack_synced.png'
+
+import slack_channel from 'images/slack_channel.png'
+
+While not everyone you work with is a New Relic user, most of your teammates are probably on Slack. Now you can discuss New Relic data with everyone and keep all insights synced between Slack and New Relic discussions.
+
+Unlike many Slack integrations that just share an update to Slack in one-direction, conversation threads on New Relic are synced in both directions with threads on Slack. When you share a question, observation, or request for assistance from New Relic to Slack, your teammates on Slack—whether they are New Relic users themselves or not—can reply to the post on Slack and these replies will be brought back into the thread on New Relic. Likewise, replies added to the thread on New Relic will be added to the thread on Slack. This back and forth sync lets you include more people in the conversation, capture the discussion along with the context in New Relic, use data to inform decisions, and build institutional knowledge. All at the same time.
```

Comment:
Really like the style of the first two paragraphs. You may want to merge the last sentence in with the previous one but overall this is super-good